mpmp/src/cmds/Prison.java

```diff
@@ -15,34 +16,33 @@
 	public void exec(String line, Conn conn) {
 		String[] args = line.split(" ");
 		Player p;
-		
+
 		if (args.length < 3) {
 			conn.sendErr(ErrCode.Usage, "prison <enter|leave> <Spieler>");
 			return;
 		}
-		
+
 		p = Player.search(String.join(" ", Arrays.copyOfRange(args, 2, args.length)));
 		if (p == null) {
 			conn.sendErr(ErrCode.NoSuchPlayer);
 			return;
 		}
-		
+
 		switch (args[1]) {
-		case "enter":
-			p.prison(true);
-			break;
-		case "leave":
-			p.prison(false);
-			break;
-		default:
-			conn.sendErr(ErrCode.Usage, "prison <enter|leave> <Spieler>");
-			return;
+			case "enter":
+				p.prison(true);
+				break;
+			case "leave":
+				p.prison(false);
+				break;
+			default:
```

Comment:
**Note:** The `case`s should be on the same indentation level as the `switch`. There's an option in NetBeans to achieve this in the Tools/Options/Formatting panel on the lower left side, IIRC.